data/abilities.ts

```diff
@@ -70,7 +70,7 @@ export const Abilities: {[abilityid: string]: AbilityData} = {
 		name: "Aftermath",
 		onDamagingHitOrder: 1,
 		onDamagingHit(damage, target, source, move) {
-			if (move.flags['contact'] && !target.hp) {
+			if (this.checkMoveMakesContact(move, source, target) && !target.hp) {
```

Comment:
I think the order is wrong here, you want to check that the target is healthy before its protective pads can protect it.